### ISSUE TITLE
Correlations tool fails with a large corpus

### ISSUE THREAD

**Issue by susanmonnelly** [sgsinclair/Voyant#471]
https://voyant-tools.org/tool/Correlations?corpus=288cf2a4af5b205c07ab68c8294e725a&view=correlations

This is an XML document that is about 29 MB in size

Although slow to load it works in other tools .

**Comment by sgsinclair** [comment]
Right, I'm not entirely sure what to do for this case since an open-ended (i.e. no query) request for correlations has to compare every word to every other word in gigantic matrix. I'm tempted to say that the tool only works for either 1) a query or 2) some threshold of top frequency terms. Any thoughts on this?

**Comment by susanmonnelly** [comment]
I'm not sure what to do either but could you at least inform the user there's a problem and instruct them how to fix it?   

 Here's a screen shot 

![Screen Shot 2020-04-02 at 10 57 02 AM](https://user-images.githubusercontent.com/12297503/78264085-be1fdc00-74d0-11ea-99e2-62231b754104.png)


I have also seen a similar problem in the Phrases tool specifically  when select different words in the drop down

![Screen Shot 2020-04-02 at 10 59 13 AM](https://user-images.githubusercontent.com/12297503/78264399-1f47af80-74d1-11ea-9edc-57ae100f3666.png)
